lib/coradoc/document/inline/anchor.rb

```diff
@@ -0,0 +1,39 @@
+module Coradoc
+  module Document
+    module Inline
+      class Anchor
+        attr_reader :id, :href, :title, :name
+
+        def initialize(options = {})
+          @id = options.fetch(:id,nil)
+          @href = options.fetch(:href,nil)
+          @title = options.fetch(:title, nil)
+          @name = options.fetch(:name,nil)
+        end
+
+        def to_adoc
+          if /^_Toc\d+$|^_GoBack$/.match @id
```

Comment:
This is odd and should not be here.

Reply:
This is copied from the other gem which was handling some odd situation. Let’s remove it.